src/lib/Scenes/Onboarding/OnboardingLogin.tsx

```diff
@@ -39,86 +39,107 @@ export const OnboardingLoginForm: React.FC<OnboardingLoginProps> = ({ navigation
   const passwordInputRef = useRef<Input>(null)
   const emailInputRef = useRef<Input>(null)
 
+  /**
+   * When we land on OnboardingLogin from the OnboardingCreatAccount
+   * withFadeAnimation is set to true therefore if the user presses
+   * on the back button to navigate to the welcome screen, the screen
+   * fades instead of translating horizontally. To avoid that we need
+   * to overwrite withFadeAnimation param once the screen shows up
+   */
+  useEffect(() => {
+    setTimeout(() => {
+      navigation.setParams({ withFadeAnimation: false })
+    }, 1000)
+    if (route.params?.email) {
+      handleChange("email")(route.params?.email || "")
```

Comment:
```suggestion
      handleChange("email")(route.param.email)
```
the truthiness was already verified in the if condition above